vissl/losses/barlow_twins_loss.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import pprint
+
+import torch
+from classy_vision.generic.distributed_util import get_cuda_device_index
+from classy_vision.losses import ClassyLoss, register_loss
+from torch import nn
+from vissl.utils.hydra_config import AttrDict
+
+
+@register_loss("barlow_twins_loss")
+class BarlowTwinsLoss(ClassyLoss):
+    """
+    This is the loss which was proposed in the Barlow Twins https://arxiv.org/abs/2103.03230v1
+    paper.
+    See the paper for the details on the loss.
+
+    Config params:
+        lambda_ (float): weight on the off-diagonal terms. It controls the trade-off between
```

Comment:
if this is config parameter, do we require `_` in `lambda_`  ? or can we rename it `lambda` ?

Reply:
Yes as `lambda` is a reserved name in Python.

Reply:
I also think `lambda_` is ugly but I didn't find a better alternative =(

Reply:
ah gotcha! I think we can go with `lambda_` for now since that way we will have a good correspondence with paper.